### ISSUE TITLE
Can't install 4.0.0-beta.2

### ISSUE THREAD

**Issue by AndrewMac** [nystudio107/craft3-recipe#56]
### Describe the bug
I do a composer install. The process crashes when it starts to generate optimized autoload files.
Error messages follows:
```
Generating optimized autoload files
94 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Exception 'Error' with message 'Class "nystudio107\recipe\FeedMeFields" not found'

in /app/vendor/nystudio107/craft-recipe/src/Recipe.php:96

Stack trace:
#0 /app/vendor/yiisoft/yii2/base/BaseObject.php(109): nystudio107\recipe\Recipe->init()
#1 /app/vendor/yiisoft/yii2/base/Module.php(161): yii\base\BaseObject->__construct(Array)
#2 /app/vendor/craftcms/cms/src/base/Plugin.php(114): yii\base\Module->__construct('recipe', Object(craft\console\Application), Array)
#3 /app/vendor/nystudio107/craft-recipe/src/Recipe.php(59): craft\base\Plugin->__construct('recipe', Object(craft\console\Application), Array)
#4 [internal function]: nystudio107\recipe\Recipe->__construct('recipe', Object(craft\console\Application), Array)
#5 /app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#6 /app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('nystudio107\\rec...', Array, Array)
#7 /app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('nystudio107\\rec...', Array, Array)
#8 /app/vendor/craftcms/cms/src/Craft.php(53): yii\BaseYii::createObject(Array, Array)
#9 /app/vendor/craftcms/cms/src/services/Plugins.php(946): Craft::createObject(Array, Array)
#10 /app/vendor/craftcms/cms/src/services/Plugins.php(228): craft\services\Plugins->createPlugin('recipe', Array)
#11 /app/vendor/craftcms/cms/src/base/ApplicationTrait.php(1471): craft\services\Plugins->loadPlugins()
#12 /app/vendor/craftcms/cms/src/console/Application.php(52): craft\console\Application->_postInit()
#13 /app/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#14 /app/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#15 /app/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct(Array)
#16 [internal function]: yii\console\Application->__construct(Array)
#17 /app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#18 /app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\console\\A...', Array, Array)
#19 /app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\console\\A...', Array, Array)
#20 /app/vendor/craftcms/cms/src/Craft.php(53): yii\BaseYii::createObject(Array, Array)
#21 /app/vendor/craftcms/cms/bootstrap/bootstrap.php(239): Craft::createObject(Array)
#22 /app/vendor/craftcms/cms/bootstrap/console.php(40): require('/app/vendor/cra...')
#23 /app/craft(12): require('/app/vendor/cra...')
#24 {main}
```
### To reproduce
Steps to reproduce the behaviour:
1. Run composer install
2. See error message above
3. Any other PHP craft actions will result in the same error
4. Need to remove recipe

### Versions
- Plugin version: 4.0.0-beta.2
- Craft version: 4.0.0-beta.4

Thanks for any time you are able to put towards looking into this, but I understand it's not one of your paid plugins

**Comment by khalwat** [comment]
Fixed in Recipe 4.0.0-beta.3 -> https://github.com/nystudio107/craft-recipe/releases/tag/4.0.0-beta.3

**Comment by AndrewMac** [comment]
Thanks so much! :-)
